Makefile

```diff
@@ -35,10 +34,10 @@ po/$(PACKAGE_NAME).js.pot:
 		--keyword=gettextCatalog.getString:1,3c --keyword=gettextCatalog.getPlural:2,3,4c \
 		--from-code=UTF-8 $$(find src/ -name '*.js' -o -name '*.jsx')
 
-po/$(PACKAGE_NAME).html.pot: $(NODE_MODULES_TEST) $(COCKPIT_REPO_STAMP)
+po/$(PACKAGE_NAME).html.pot: package-lock.json $(COCKPIT_REPO_STAMP)
```

Comment:
I liked the `NODE_MODULES_TEST` constant to express the meaning of a dependency and to avoid repetition. But no strong opinion.

Reply:
I more or less copied the rule directly from the cockpit project, and it really absolutely needs to be this file for it to work with the existing script...

I feel like depending on `package-lock.json` is a pretty clear indication of what's going on here.